### PR TITLE
Fix tslint and linting errors

### LINKED ISSUE #57: Linting does not check all files
tslint only checks the files that are in the src and tests directory, but no sub-directories!
See [package.json](https://github.com/phovea/phovea_ui/blob/0212fdacbe6efce71be3115fcbc8d520530b95e0/package.json#L34)

### PR DESCRIPTION
**Pull request by keckelt**
Closes #57 

Update tslint & fix linting errors

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
   },
   "scripts": {
     "compile": "tsc",
-    "lint": "tslint -c tslint.json src/**.ts tests/**.ts",
+    "lint": "tslint -c tslint.json -p .",
```
Comment (thinkh) [inline package.json]: What does the flag `-p` stand for?

Comment (keckelt) [reply package.json]: It tells tslint to start in the current directory

```
-p, --project:
    The path or directory containing a tsconfig.json file that will be
    used to determine which files will be linted. This flag also enables
    rules that require the type checker.
```
https://palantir.github.io/tslint/usage/cli/

**Comment by keckelt** [comment]
Yes -  i posted a list in the infrastructure chat.

**Comment by thinkh** [comment]
I close this PR in favor of https://github.com/phovea/phovea_ui/pull/81, which integrates your changes. Thanks for fixing this bug initially. 👍

**Pull request closed without merging**